source/components/triggering-changes-with-actions.md

```diff
@@ -156,3 +156,56 @@ a function instead.
 Actions in components allow you to
 decouple an event happening from how it's handled, leading to modular,
 more reusable components.
+
+## Calling Actions Across Component Layers
+
+When your components go multiple layers deep, its common to need to handle an action several layers up the tree.
+Using action helpers, its possible to make top level actions available at the bottom layers of your component tree
```

Comment:
"the action helper"
"it's possible"

I'd suggest rephrasing "top level actions". "top level" is a bit fuzzy for more beginner readers.